py3status/modules/volume_status.py

```diff
@@ -18,6 +20,9 @@
         (default 10)
     channel: Alsamixer channel to track.
         (default 'Master')
+    color_muted: set to 1 to color 'format_muted' depending on the volume
+        percent.
+        (default 0)
```

Comment:
Indeed. I forgot. Anyhow, per your previous comment, I'll reserve this for another PR once this one is done.